README.md

```diff
@@ -145,12 +145,12 @@ and the language:
     -   Operator overloading
     -   Lexical and syntactic structure
     -   Code organization and modular structure
--   A prototype interpreter demo that can both run isolated examples and gives a
-    detailed analysis of the specific semantic model and abstract machine of
-    Carbon. We call this the [Carbon Explorer](/explorer/).
 -   An under-development [compiler and toolchain](/toolchain/) that will compile
     Carbon (and eventually C++ code as well) into standard executable code. This
     is where most of our current implementation efforts are directed.
+    -   Historically, there was also a prototype
+        [explorer](https://github.com/carbon-language/explorer) interpreter that
+        is no longer under development, and has been archived.
```

Comment:
Optional: maybe also point out that explorer doesn't implement the current Carbon language design.
```suggestion
        [explorer](https://github.com/carbon-language/explorer) interpreter that
        implemented an older version of the Carbon language design, but is no
        longer under development and has been archived.
```